tsconfig.json

```diff
@@ -21,6 +21,7 @@
     "**/jest-coverage/**/*",
     "**/__mocks__/**/*",
     "**/storybook-build/**/*",
+    "**/*.stories.*",
```

Comment:
Even though we allow it ignore check for now so we can still write stories for the time being

Reply:
Specifically ignoring `*.stories.tsx` for the time being because it will not allow us to write any stories that don't use a typescript component or a component that has types.